docs/sources/release-notes/v2.1.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Grafana Mimir version 2.1 release notes"
+menuTitle: "V2.1 release notes"
+description: "Release notes for Grafana Mimir version 2.1"
+weight: 200
+---
+
+# Grafana Mimir version 2.1 release notes
+
+Grafana Labs is excited to announce version 2.1 of Grafana Mimir, the most scalable, most performant open source time series database in the world. 
+
+Below we highlight the top features, enhancements and bugfixes in this release, as well as relevant callouts for those upgrading from Grafana Mimir 2.0. The complete list of changes is recorded in the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Mimir on ARM**: The `mimir build` command now builds both amd64 and arm64 Docker images, making it easier for those on arm-based machines to develop and run Mimir. Both amd64 and arm64 images will now be published to the [Mimir docker registry](https://hub.docker.com/r/grafana/mimir). Note that our existing integration test suite only uses the amd64 images, which means we cannot make any functional or performance guarantees about the arm64 images.  
```

Comment:
We don't have `mimir build` command. There is `make build`, but that only builds local binary. It is possible to use `make push-multiarch-mimir` to publish multiplatform image to Docker Hub, but it requires credentials. In the end, I would suggest to not call out any specific command.

```suggestion
- **Mimir on ARM**: We now publish Docker images for both `amd64` and `arm64`, making it easier for those on arm-based machines to develop and run Mimir. Multiplaform images are available from the the [Mimir docker registry](https://hub.docker.com/r/grafana/mimir). Note that our existing integration test suite only uses the `amd64` images, which means we cannot make any functional or performance guarantees about the `arm64` images.  
```

Reply:
@pstibrany -- Should I mention that you can locally build arm64 images too (even if you can't push them to docker hub)?

Reply:
Accepted your suggestion in the meantime.

Reply:
Perfect - thanks for clarifying! I will leave the release notes as written then.